scripts/microbench/microbench.sh

```diff
@@ -1,17 +1,22 @@
 export LD_PRELOAD=${CONDA_PREFIX:-"$(dirname $(which conda))/../"}/lib/libiomp5.so:${CONDA_PREFIX:-"$(dirname $(which conda))/../"}/lib/libjemalloc.so
-export MALLOC_CONF="oversize_threshold:1,background_thread:true,metadata_thp:auto,dirty_decay_ms:-1,muzzy_decay_ms:-1"
+export MALLOC_CONF="oversize_threshold:1,background_thread:true,metadata_thp:auto,dirty_decay_ms:9000000000,muzzy_decay_ms:9000000000"
```

Comment:
why we need change back to old config for jemalloc?

Reply:
micro bench script use 9000000000 config continuously, also update in microbench?

Reply:
sure, we need align performance test BKC